samples/sandwiches/Makefile

```
@@ -0,0 +1,28 @@
+SWIFT_BIN = ../../bin/swift/bin
+SWIFT_LIB = ../../bin/swift/lib/swift/macosx/
+
+
+SWIFTC = $(SWIFT_BIN)/swiftc
+SDK = `xcrun --sdk macosx --show-sdk-path`
+SWIFTARGS = -v -sdk $(SDK) -L $(SWIFT_LIB) -emit-module -emit-library
+
+BINDINGTOOLSFORSWIFT=../../binding-tools-for-swift
+
+OUTPUT_MODULE=Sandwiches
+
+libSandwiches.dylib: *.swift 
+	@rm -rf XamWrappingSource armv7 armv7s arm64 x86_64
+	@mkdir XamGlue.framework
+	@cp ../../lib/SwiftInterop/mac/XamGlue.framework/XamGlue .
```

Is the `XamGlue` binary supposed to go into the current directory, or into `XamGlue.framework`?

```suggestion
	@cp ../../lib/SwiftInterop/mac/XamGlue.framework/XamGlue XamGlue.framework/
```

---

current directory.